src/site/includes/breadcrumbs.drupal.liquid

```diff
@@ -22,7 +17,7 @@
           onclick="recordEvent({ 'event': 'nav-breadcrumb', 'breadcrumb-click-label': '{{ crumb.text }}', 'breadcrumb-click-level': '{{ forloop.index }}', 'breadcrumb-total-levels': '{{ crumbs.length }}' });" 
           aria-current="page" 
           href="/{{ path }}"
-          lang={{lang}}
+          lang={{ entityUrl.path | detectLang }}
```

Comment:
I don't think you need surrounding quotes here but might be nice for consistency with the other attributes. Same idea with the other changes too.

```suggestion
          lang="{{ entityUrl.path | detectLang }}"
```